packages/magika.vm/magika.vm.nuspec

```
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>magika.vm</id>
-    <version>0.0.0.20240607</version>
+    <version>0.0.0.20241124</version>
```

Installing a concrete version is a good idea and now we can use that version for the package version too:
```suggestion
    <version>0.5.0</version>
```